virtual_agent:
  position: {'x': 0.0, 'y': 0.0, 'z': 0.0}
  translation_x: 0.0
  translation_y: 0.0
  translation_z: 0.0
crazyflie1:
  off_set: {'x': 0.25, 'y': 0.0, 'z': 0.5}
  translation_agent_x: 0.0
  translation_agent_y: 0.0
  translation_agent_z: 0.0
  takeoff: 1
  land: 0
crazyflie2:
  off_set: {'x': -0.25, 'y': 0.5, 'z': 0.5}
  translation_agent_x: 0.0
  translation_agent_y: 0.0
  translation_agent_z: 0.0
  takeoff: 1
  land: 0
crazyflie3:
  off_set: {'x': -0.25, 'y': -0.5, 'z': 0.5}
  translation_agent_x: 0.0
  translation_agent_y: -0.0
  translation_agent_z: 0.0
  takeoff: 1
  land: 0
scaling: 1.0
rotation_x: 0.0
rotation_y: 0.0
rotation_z: 45.0
